### ISSUE TITLE
[BUG] Excel.Table does not expose correct properties

### ISSUE THREAD

**Issue by DPWalther** [exceljs/exceljs#1904]
## 🐛 Bug Report

Excel.TableProperties does define a propertie columns and with that does Excel.Table as it's an extension of Excel.TableProperties.

But this propertie doesn't actually exist on Excel.Table. In reality this propertie is on table.table.columns

Lib version: 4.3.0

## Steps To Reproduce

This should be valid code by how the interface is currently written. (columns propertie beeing present on Excel.Table via. the extension of Excel.TableProperties which defines columns as a propertie)

```typescript
const wb = await new Excel.Workbook().xlsx.readFile("MyExcel.xlsx");
const ws = wb.getWorksheet("MyWorksheet");
const table = ws.getTable('MyTable');
console.log(table.columns);
```

But table.columns doesn't exist. To access it, you have to use the following code
```typescript
const wb = await new Excel.Workbook().xlsx.readFile("MyExcel.xlsx");
const ws = wb.getWorksheet("MyWorksheet");
const table = ws.getTable('MyTable');
console.log(table.table.columns);
```

or if you don't want a linter error:
```typescript
const wb = await new Excel.Workbook().xlsx.readFile("MyExcel.xlsx");
const ws = wb.getWorksheet("MyWorksheet");
const table = ws.getTable('MyTable');
console.log((table as any).table.columns);
```

## The expected behaviour:

Either the columns propertie should return the correct value directly on the table or an addiontal propertie table which contains columns should be defined.

**Comment by skypesky** [comment]
@DPWalther 
Can you provide a sample excel file and sample file?

**Comment by free-lunch** [comment]
I have the same issue.

**Comment by GaryHyland** [comment]
That prob explains why I get a corrupted XLSX if I try and use sheet.addTable(...).getColumns(1).width = 15 also.

**Comment by NiklasPor** [comment]
Yes, it seems like `addTable` returns an object containing the worksheet and table, instead of just the table:

```ts
  const table = sheet.addTable({
    name: st('entities.components'),
    ref: 'A2',
    columns: [
      { name: 'Label', filterButton: true },
      { name: nameColumnKey, filterButton: true },
    ],
    rows: [],
    headerRow: true,
    style: { theme: 'TableStyleLight1' },
  });

  console.log(table);
  ```

![image](https://github.com/exceljs/exceljs/assets/13211347/91f33a8b-c90e-4cce-ae15-a74ebacdaad9)

**Comment by dimbslmh** [comment]
> Yes, it seems like `addTable` returns an object containing the worksheet and table, instead of just the table

https://github.com/exceljs/exceljs/blob/5bed18b45e824f409b08456b59b87430ded023ab/lib/doc/worksheet.js#L794-L810

https://github.com/exceljs/exceljs/blob/5bed18b45e824f409b08456b59b87430ded023ab/lib/doc/table.js#L69-L79

It seems to me that the getTable and removeTable are also broken because of this issue.

Possible fix, not sure if things will break, **this should be thoroughly tested**:
```js
addTable(model) {
  const instance = new Table(this, model);
  const { table ) = instance;
  this.tables[model.name] = table;
  return table;
}
```
